app/presenters/event_presenter.rb

```diff
@@ -15,7 +15,9 @@ def as_json(options = {})
       description: description,
       state_params: event.state_params,
       user: user,
-      unlinkable: event.unlinkable
+      unlinkable: event.unlinkable,
+      organization: event.organization ? OrganizationIndexPresenter.new(event.organization) : nil,
```

Comment:
This line will add a `select * from organizations where organization.id = event.organization_id` to every single instance of the event presenter. We likely want to look at places where we're calling this and add an `eager_load` or `include` to the initial query to prevent that or we'll get an extra db query for every row in the table.

Reply:
I found a few places that uses this presenter under the hood and added `:organization` in their already existing `includes` statements.